kuma/static/js/payments-confirmation.js

```diff
@@ -28,11 +28,11 @@
             sessionStorage.removeItem(amountSubmittedStoreKey);
         });
 
-    } else if (path.includes('/payments/recurring/success')) {
+    } else if (path.includes('/payments/recurring/success') && win.mdn.features.localStorage) {
         mdn.analytics.trackEvent({
             category: 'Recurring payments',
             action: 'success',
-            label: originalUserAuth,
+            label: originalUserAuth || 'lost',
```

Comment:
So, just to be sure I understand this correctly. If we are on either the success or error page, and `localStorage` is enabled, we send the event. If for whatever reason `originalAuth` is `undefined`, we instead the label to "lost".

So this essentially means the user's `localStorage` was disabled during auth, but was then enabled post payment?

Reply:
yep! Success/error pages aren't guarded, so we just check local storage to confirm the user is coming from the payment flow to prevent extra events being fired on refresh.
'lost' in theory should never be sent if all users follow the expected flow, this is just a case to help us debug a problem is there is one.